### PR TITLE
adds Dockerfile

### PR DESCRIPTION
**Pull request by breign**
For easier deployment on different machines and environments, I'm proposing a Dockerfile.

### CODE REVIEW

File: contrib/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# example of Dockerfile that builds release of electrumx-1.13.0
+# ENV variables can be overrided on the `docker run` command
+
+FROM ubuntu:18.04
+
+WORKDIR /
+ADD https://github.com/kyuupichan/electrumx/archive/1.13.0.tar.gz /
```
Comment (streetcrypto7) [inline contrib/Dockerfile]: 🤔 Could we change this to something like git clone https://github.com/kyuupichan/electrumx?

File: contrib/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+RUN tar zxvf *.tar.gz
+
+RUN apt-get update && \
+        apt-get -y install python3.7 python3-pip librocksdb-dev libsnappy-dev libbz2-dev libz-dev liblz4-dev && \
+        pip3 install aiohttp pylru python-rocksdb
+
+RUN cd /electrumx* && python3 setup.py install
+
+ENV SERVICES="tcp://:50001"
+ENV COIN=BitcoinSV
```
Comment (streetcrypto7) [inline contrib/Dockerfile]: BitcoinSV -> BitcoinSegwit please

File: contrib/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+ENV DB_DIRECTORY=/db
+ENV DAEMON_URL="http://username:password@hostname:port/"
```
Comment (streetcrypto7) [inline contrib/Dockerfile]: lmao where is the bitcoind?

File: contrib/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+ENV DB_ENGINE=rocksdb
+ENV MAX_SEND=10000000
+ENV BANDWIDTH_UNIT_COST=50000
+ENV CACHE_MB=2000
+
+VOLUME /db
+
+RUN mkdir -p "$DB_DIRECTORY" && ulimit -n 1048576
+
+CMD ["/usr/bin/python3", "/usr/local/bin/electrumx_server"]
```
Comment (streetcrypto7) [inline contrib/Dockerfile]: Hmm shouldn't be /usr/bin/python3.7 instead of /usr/bin/python3?

File: contrib/Dockerfile (hunk 3 above)
Comment (breign) [reply contrib/Dockerfile]: ENV can also be set at `docker run` command to override anything to their needs.

File: contrib/Dockerfile (hunk 4 above)
Comment (breign) [reply contrib/Dockerfile]: Ubuntu distro has a symlink to their python 3.* installation.

File: contrib/Dockerfile (hunk 2 above)
Comment (breign) [reply contrib/Dockerfile]: This is all configurable at run.

File: contrib/Dockerfile (hunk 1 above)
Comment (breign) [reply contrib/Dockerfile]: Please do. I was just pasting a Dockerfile that works for me.

**Pull request merged**